services/ui-src/src/components/pages/Profile/ProfilePage.test.tsx

```diff
@@ -27,22 +29,25 @@ describe("Test ProfilePage for admin users", () => {
     render(ProfilePageComponent);
   });
   test("Check that Profile page renders properly", () => {
-    expect(screen.getByTestId("profile-view")).toBeVisible();
+    const rows = screen.getAllByRole("row");
+    expect(rows[0]).toHaveTextContent("adminuser@test.com");
   });
 
   test("Check that there is an banner editor button visible", () => {
-    expect(screen.getByTestId("banner-admin-button")).toBeVisible();
+    const buttons = screen.getAllByRole("button");
+    expect(buttons[0]).toHaveTextContent("Banner Editor");
   });
 
   test("Check that the state field is set to N/A", () => {
     expect(screen.getByText("State")).toBeVisible();
     expect(screen.getByText("N/A")).toBeVisible();
   });
 
-  test("Check that admin button navigates to /admin on click", () => {
-    const adminButton = screen.getByTestId("banner-admin-button");
-    expect(adminButton).toBeVisible();
-    fireEvent.click(adminButton);
+  test("Check that admin button navigates to /admin on click", async () => {
+    await act(async () => {
+      const adminButtons = screen.getAllByRole("button");
+      await userEvent.click(adminButtons[0]);
+    });
```

Comment:
I think we can remove the `act()` here, but I think we can also improve the query. Hopefully this button has a unique `name` we can pass to `getByRole`?

Reply:
It doesn't have a unique name. "Banner Editor" appears twice.

Reply:
removing 'act' causes the test to fail and it doesn't find the path "/admin"

Reply:
We have two buttons named Banner Editor?? That seems really odd 👀

Reply:
I think it was a fluke. I updated the tests. It's not showing up twice anymore.